rust/noosphere-cli/src/native/cli.rs

```diff
@@ -61,6 +61,11 @@ pub enum OrbCommand {
         /// The port that the gateway should listen on
         #[clap(short, long, default_value = "4433")]
         port: u16,
+
+        /// If set, the amount of memory that the storage provider may use
+        /// for caching in bytes.
+        #[clap(long)]
```

Comment:
Can we get a short arg for this? e.g., `short = 'S'` or something

Reply:
Punting on this for now, as this is a low-level operator command